src/Bridge/Faker/Provider/StoryblokProvider.php

```diff
@@ -505,6 +506,39 @@ public function assetResponse(array $overrides = []): array
         );
     }
 
+    /**
+     * @param array{
+     *     id?: string,
+     *     url?: string,
+     *     email?: string,
+     *     linktype?: string,
+     *     fieldtype?: string
+     * } $overrides
+     *
+     * @return array{
+     *     id: string,
+     *     url: string,
+     *     email: string,
+     *     linktype: string,
+     *     fieldtype: string
+     * }
+     */
+    public function multiLinkResponse(array $overrides = []): array
```

Comment:
I would prefer to use foundry here

Reply:
Thats what we can add in addition. but this package provides a faker extension